native/NativeSampleApps/SmartSyncExplorer/src/com/salesforce/samples/smartsyncexplorer/ui/MainActivity.java

```diff
@@ -132,6 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
 	}
 
 	@Override
+	protected void logoutCompleteActions() {
+		super.logoutCompleteActions();
```

Comment:
If there is a revoke, the client manager will call logout without passing a front activity and without requesting login to be shown. So the login screen will only be shown the next time the user changes activities.